test/data/prepend_steps/execution-environment.yml

```diff
@@ -1,5 +1,14 @@
 ---
-version: 1
+version: 3
+
+images:
+  base_image:
+    name: quay.io/centos/centos:stream9
+
 additional_build_steps:
-  prepend: |
-    RUN whoami
+  prepend_base: |
+    RUN echo "hello from prepend_base" > /baseout.txt
```

Comment:
If we do not intend to test v1 `prepend` anymore (we'd be losing coverage if we change this), then we don't need this test data (or the test that uses it) anymore since all of the prepend/append steps are tested in `test/data/v3/pre_and_post/ee.yml`.

Reply:
I think we can probably move most of the v1 tests like that to unit tests and/or create tests- v1 builds are by definition going to be *much* more expensive, and the value of a full build integration test that ensures that a line appears in the output probably isn't worth the time spent on it.

Reply:
(moved to a `create` test and killed the test data)